src/bp/core/services/hook/hook-service.ts

```diff
@@ -13,103 +13,62 @@ import { TYPES } from '../../types'
 import { VmRunner } from '../action/vm'
```

Comment:
I think it would be nice to keep some of the variable, like using `const args = {} ... super(1000, args, ...) `, since it's a bit quicker to see what they are used for, and if there's multiple arguments, it's easier

Reply:
yeah we have syntax highlight for that though